### PR TITLE
Auto-formatting, warning fix and small refactor.

### PR DESCRIPTION
**Pull request by XericZephyr**
- Fix the "op never used" warning.
- Refactor the reader to potentially support label_weight output.

### CODE REVIEW

**Comment by XericZephyr** [comment]
Verify the CLA status @googlebot

File: readers.py [hunk 1]
```diff
@@ -208,48 +218,67 @@ def prepare_reader(self,
...
-          self.max_frames,
-          max_quantized_value,
-          min_quantized_value)
+          self.feature_sizes[feature_index], self.max_frames,
+          max_quantized_value, min_quantized_value)
       if num_frames == -1:
         num_frames = num_frames_in_this_feature
       else:
-        tf.assert_equal(num_frames, num_frames_in_this_feature)
+        tf.assert_equal(num_frames, num_frames_in_this_feature).mark_used()
```
Comment (LeegleechN) [inline readers.py]: I think this is never used because it isn't a required node in the computational graph.

Comment (XericZephyr) [reply readers.py]: Removed.

**Comment by XericZephyr** [comment]
Addressed. PTAL.

**Pull request merged**